src/app/games/new-game/NewGameRules.ts

```diff
@@ -29,13 +32,40 @@ export class NewGameNode extends GameNode<NewGameMove, NewGameState> {}
  * It should be a singleton class.
  * It is used by the wrappers to check the legality of a move, and to apply the move on a state.
  */
-export class NewGameRules extends Rules<NewGameMove, NewGameState, NewGameLegalityInfo> {
+export class NewGameRules extends Rules<NewGameMove, NewGameState, RulesConfig, NewGameLegalityInfo> {
 
     /**
      * This is the singleton instance. You should keep this as is, except for adapting the class name.
      */
     private static singleton: MGPOptional<NewGameRules> = MGPOptional.empty();
 
+    /**
+     * If you want your game to be configurable (different board sizes by example)
```

Comment:
for example
le "Here" juste en dessous est la suite de la phrase, pas le majusculer comme ça !
Puis mettre des points à la fin des phrases ici

Reply:
poucé mais pas poussé !